### PR TITLE
Motech 1625 Update Angular JS library to version 1.5

### PR DESCRIPTION
**Pull request by wstrzelczyk**

### CODE REVIEW

File: 0002-dodatkowe-klasy-email-dla-testowania-many-to-many.patch [hunk 1]
```diff
@@ -0,0 +1,188 @@
+From 8be733af533a982af1a1c5382b47db5f0ec9f553 Mon Sep 17 00:00:00 2001
```
Comment (sebbrudzinski) [inline 0002-dodatkowe-klasy-email-dla-testowania-many-to-many.patch]: Remove this from the pull request

**Comment by sebbrudzinski** [comment]
@pgesek Can you assign this to one of the testers?

**Comment by PiotrZurawskiSolDevelo** [comment]
Checked, all errors are in this file:
[Errors in pullRequest.pdf](https://github.com/motech/motech/files/261335/Errors.in.pullRequest.pdf)

**Comment by PiotrZurawskiSolDevelo** [comment]
Additionally when I want to create Schedule Job i get undefined error. Error not appear on MOTECH in version 0.29.

**Comment by wstrzelczyk** [comment]
In my opinion all errors reported in attached document are not connected with my changes:
Ad1.
https://applab.atlassian.net/browse/MOTECH-2473
Ad2.
This problem is connected with others modules (metrics) or not cleaning old data in .motech folder
Ad3. 
I do not noticed, see Ad2

and also I had not problem with creating Schedule Job.

**Comment by PiotrZurawskiSolDevelo** [comment]
@wstrzelczyk I have not installed metrics module in my MOTECH application, additionally I always when I build MOTECH application I delete .motech folder. 
For first error I agree.

**Comment by wstrzelczyk** [comment]
@PiotrZurawskiSolDevelo  You do not need install metrics module to crashed MOTECH!
Tray build core modules without other modules.

**Comment by wstrzelczyk** [comment]
"test this please"

**Comment by PiotrZurawskiSolDevelo** [comment]
Checked, I spent bit more on this ticket. 
Tickets which I reported before are not depended with Your changes.
I found only one more issue which I didn't find before:
When I click in CSD Module into Show or Download button I get error in console. I Checked this also on MOTECH 0.29 with all modules and this error is not appear:
csd show and download button error

**Comment by wstrzelczyk** [comment]
@PiotrZurawskiSolDevelo I added new changes for CSD module:
https://github.com/motech/modules/pull/285

**Comment by PiotrZurawskiSolDevelo** [comment]
@wstrzelczyk I checked this with Your fix, now it's possible to use this buttons.
Now it's looking like everything is working well.

**Pull request merged**